po/ja.po

```diff
@@ -16663,16 +16669,24 @@ msgid ""
 "contain `Send` and `Sync` types. You can also implement them manually when "
 "you know it is valid."
 msgstr ""
+"`Send`と`Sync`は[unsafeなトレイト](../unsafe/unsafe-traits.md)です。 あなたが"
+"新たに定義する型が`Send`と`Sync`の型のみを含む場合、コンパイラはその新しい型"
+"に対して`Send`と`Sync`を自動的に導出します。また、そうでなくても、もし妥当で"
```

Comment:
> また、そうでなくても、もし妥当で

This can be rewrite as "そうでなくても妥当で" to remove redundancy. 

In my understanding, "そうでなくても" has the meaning of assumption as "もし"